docs/features/macros.md

```
@@ -4,91 +4,154 @@ hide:
+| Primary color hue | 200 |
+| Segment N opacity | 0-32 |
 
-This potentiometer should be supplied 3.3V and GND, with it's output supplied to A0 (or any other ADC pin you specify), recommended 10KΩ or greater. 
+Connect the potentiometer to 3.3 V and GND. Connect its output to A0, or to the ADC pin that you configure. Use a potentiometer of 10 kΩ or greater.
 
-!!! info "Do not use ESP32 ADC2 GPIO pins for analog buttons"
+!!! info "Do Not Use ESP32 ADC2 GPIO Pins for Analog Buttons"
     On ESP8266, you can only have a single analog button on pin A0, the pin set in the settings UI is ignored.  
     On ESP32, only ADC1 pins will work for analog input while WiFi is active (pins 32-39). ADC2 pins will not work.
 
 #### Global Brightness
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)